### ISSUE TITLE
Have an official GitHub Action for invoking tox

### ISSUE THREAD

**Issue by webknjaz** [tox-dev/gh-action-tox#1]
The motivation is similar to https://discuss.python.org/t/official-github-action-for-publishing-to-pypi/1061.
I think, providing a ready-to-use action for executing some env (of a bunch of them) is nice.

It could post results back to GitHub via Checks API but I'm not sure whether there's any generic/parsable output of tox execution to be used for this.

@gaborbernat let's discuss this IRL.

**Comment by webknjaz** [comment]
A very minimalistic example of such action: https://github.com/sanitizers/octomachinery/blob/master/.github/actions/python3.7-tox/Dockerfile

<blockquote><img src="https://avatars3.githubusercontent.com/u/40318227?s=400&v=4" width="48" align="right"><div><img src="https://github.githubassets.com/favicon.ico" height="14"> GitHub</div><div><strong><a href="https://github.com/sanitizers/octomachinery">sanitizers/octomachinery</a></strong></div><div>Invisible engine driving octobot machines. Simple, yet powerful. [DEMO BOT @ https://github.com/sanitizers/chronographer-github-app] | official web-site is at --> - sanitizers/octomachinery</div></blockquote>

**Comment by gaborbernat** [comment]
Looks good @webknjaz please create a PR with the code under this organization, thanks!

**Comment by gaborbernat** [comment]
Can we add some basic documentation in the README about how to use it?

We probably also want an image that has more than one python, I would propose something that has the core currently supported set (2.7, 3.5, 3.6, 3.7). In the current form, only 3.7 envs will run. Or we want a separate image per target python.

**Comment by webknjaz** [comment]
Yes, that would be great. We could also mention this in tox's docs too. I just haven't had time to add more. Also, tox-dev doesn't have Actions enabled which apparently prevents us from publishing Actions to the Marketplace. I'll try to take care of this as well.

**Comment by gaborbernat** [comment]
Any day now 👍  ![screenshot-github com-2019 05 10-13-17-10](https://user-images.githubusercontent.com/690238/57526684-f1400680-7325-11e9-9546-549b4bba5a2f.png)

**Comment by webknjaz** [comment]
I was actually going to ask a human from GitHub to do a favor :)

**Comment by njzjz** [comment]
I expect an action for tox-conda!

**Comment by gaborbernat** [comment]
We don't need a specialized version of it, just define tox-conda in the requires part to tox.ini and you'll have it auto provisioned👌

**Comment by njzjz** [comment]
But there is no conda installed in this action. It will be better if an action based on miniconda is created.

**Comment by webknjaz** [comment]
@njzjz feel free to suggest an image having all of the major Python interpreters there.

**Comment by martinlanton** [comment]
Hi!
I was looking for a github action for tox, but the only one I found was specifically for fedora : https://github.com/marketplace/actions/python-tox-on-fedora

I did find an `action` repository under tox-dev though : https://github.com/tox-dev/action
However this one is empty.

There also seem to be a `gh-action-tox` repository, which might be what I am looking for, but this one only contains one docker file, and the readme doesn't tell much... so I was wondering if I was missing something or if there was a place I could find the action mentioned in this thread, and/or some information on how to run it.
Cheers!
